doc/vxlan/data_plane_VXLAN_HLD.md

```diff
@@ -0,0 +1,507 @@
+    deactivate vxlanmgr
+
+ 
+    A->>vxlanmgr: Add VXLAN_REMOTE_VTEP_VNI_TABLE
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: Add VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate vxlanorch
+    vxlanorch->>ASIC_DB: sai_tunnel_api->create_tunnel,<br>sai_tunnel_api->create_tunnel_term,
+    vxlanorch->>portorch: addTunnel
+    portorch->>ASIC_DB: sai_bridge_api->create_bridge_port with learning enabled
+    vxlanorch->>portorch: addVlanMember
+    portorch->>ASIC_DB: sai_vlan_api->create_vlan_member
+    deactivate APP_DB
+    deactivate vxlanmgr
+    deactivate vxlanorch
+
+```
+
+##### Figure 3. Dataplane tunnel removal
+```mermaid
+%%{init: {'theme':'base', 'themeVariables':{'primaryColor': '#FFE068', 'fontFamily': 'trebuchet ", 'fontSize': '30px', "signalColor": "brown"}}}%%
+sequenceDiagram
+    participant A as CLI script
+    participant vxlanmgr
+    participant APP_DB as APP DB
+    participant vxlanorch
+    participant portorch
+    participant ASIC_DB as ASIC DB
+
+    A->>vxlanmgr: Del VXLAN_REMOTE_VTEP_VNI_TABLE
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: Del VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_REMOTE_VNI_TABLE
+    activate vxlanorch
+    vxlanorch->>portorch: removeVlanMember
+    portorch->>ASIC_DB: sai_vlan_api->remove_vlan_member
+    vxlanorch->>portorch: delTunnel
+    portorch->>ASIC_DB: sai_bridge_api->remove_bridge_port
+    vxlanorch->>ASIC_DB: sai_tunnel_api->remove_tunnel_term_table_entry,<br>sai_tunnel_api->remove_tunnel
+    deactivate APP_DB
+    deactivate vxlanmgr
+    deactivate vxlanorch
+
+    A->>vxlanmgr: Del VXLAN_REMOTE_VTEP_TABLE
+    activate vxlanmgr
+    Note right of vxlanmgr: on last entry
+    vxlanmgr->>APP_DB: Del VXLAN_DATAPLANE_VTEP_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_DATAPLANE_VTEP_TABLE
+    deactivate APP_DB
+    deactivate vxlanmgr
+
+    A->>vxlanmgr: Del VXLAN_TUNNEL_MAP
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: VXLAN_TUNNEL_MAP
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_TUNNEL_MAP
+    activate vxlanorch
+    vxlanorch->>ASIC_DB: sai_tunnel_api->remove_tunnel,<br>sai_tunnel_api->remote_tunnel_map
+    deactivate vxlanmgr
+    deactivate APP_DB
+    deactivate vxlanorch
+
+    A->>vxlanmgr: Del VXLAN_TUNNEL_TABLE
+    activate vxlanmgr
+    vxlanmgr->>APP_DB: VXLAN_TUNNEL_TABLE
+    activate APP_DB
+    APP_DB->>vxlanorch: VXLAN_TUNNEL_TABLE
+    deactivate vxlanmgr
+    deactivate APP_DB
+```
+
+
+#### 4.3.2 P2MP Tunnel Creation
+A P2MP tunnel uses a single tunnel termination. Because tunnel termination is utilized for MAC learning, supporting P2MP requires manual MAC configuration, which undermines the goal of dynamic learning.
+
+#### 4.4 Fdborch
+* FDB entries will be learned over tunnel interface and not configured via APP_VXLAN_FDB_TABLE_NAME(as in BGP-EVPN). So they will be handled as regular MAC entries.
```

Comment:
MAC provisioned by the controller will be written into the VXLAN_FDB_TABLE.

Reply:
I missed this part. VXLAN_FDB_TABLE is used to provision MAC received from EVPN.